Tools/LinearModelTraining/LinearModelTraining.swift

```diff
@@ -7,7 +7,7 @@ import MLXNN
 import MLXOptimizers
 import MLXRandom
 
-extension MLX.DeviceType: ExpressibleByArgument {
+extension MLX.DeviceType: @retroactive ExpressibleByArgument {
```

Comment:
Hrm, that seems to be a swift6 compiler feature -- have to figure out how to express this so that older compilers will accept it.